cfgov/paying_for_college/jinja2/paying-for-college/disclosure.html

```diff
@@ -1969,10 +1969,11 @@ <h2 class="step_heading">
                                     eligibility.
                                 </p>
                                 <div class="continue_controls">
-                                    <button class="a-btn a-btn__full-on-xs"
-                                    title="Continue to Step 2" type="button">
-                                    Continue to Step 2
+                                    <button class="a-btn a-btn__full-on-xs a-btn__disabled"
+                                    title="Continue to Step 2" type="button" disabled>
+                                    {{ svg_icon('updating') }} Loading program data...
                                     </button>
+                                    <div> </div>
```

Comment:
What's this?

Reply:
Oh, probably some typo or something. Removing it. :)